src/DroneSafetyApi/Models/HeatmapsQuery.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Azure.Documents.Spatial;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
The heatmaps query object is for encapsulating the query parameters for the API, and has nothing to do with databases. The Linq is there because it comes by default. We will optimise imports at a later stage.